Makefile

```diff
@@ -1,4 +1,4 @@
-SERVER_CONFIG = config/kinto.ini
+SERVER_CONFIG = config/example.ini
```

Comment:
I concur with Ethan, I would suggest to copy `example.ini` to `kinto.ini` (in the Makefile for example)

> You might also want to put config/kinto.ini in .gitignore.

+1